### PR TITLE
fixing a few things on graphics

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

File: instat/dlgScatterPlot.vb [hunk 1]
```diff
@@ -90,6 +91,7 @@ Public Class dlgScatterPlot
         ucrSelectorForScatter.Reset()
         ucrSelectorForScatter.Focus()
         ucrVariablesAsFactorForScatter.ResetControl()
+        ucrVariablesAsFactorForScatter.SetReceiverStatus()
```
Comment (dannyparsons) [inline instat/dlgScatterPlot.vb]: Why is this done in Initialise and SetDefaults? You should only need it in one of them.

Comment (dannyparsons) [inline instat/dlgScatterPlot.vb]: And isn't this done automatically in the control anyway?

Comment (maxwellfundi) [inline instat/dlgScatterPlot.vb]: @dannyparsons yeah you are right. i didnt think about it. i should remove it

Comment (dannyparsons) [inline instat/dlgScatterPlot.vb]: That would be great.

**Comment by maxwellfundi** [comment]
@dannyparsons  i fixed this already

File: instat/dlgScatterPlot.vb (hunk 1 above)
Comment (dannyparsons) [inline instat/dlgScatterPlot.vb]: It's still here on this line.

Comment (maxwellfundi) [inline instat/dlgScatterPlot.vb]: Aaaa yeah am getting it off. I thought we had said we should only use it
once.

Regards
Maxwell Fundi
|| Phone :+254728508241||

**Comment by maxwellfundi** [comment]
@dannyparsons Done!

File: instat/dlgScatterPlot.vb (hunk 1 above)
Comment (dannyparsons) [inline instat/dlgScatterPlot.vb]: Initially I thought it should be done once, but it is done automatically in the control so we never need it in the dialog.

**Pull request merged**